pkg/scheduler/algorithm/predicates/metadata.go

```diff
@@ -53,6 +53,9 @@ type predicateMetadata struct {
 	serviceAffinityInUse               bool
 	serviceAffinityMatchingPodList     []*v1.Pod
 	serviceAffinityMatchingPodServices []*v1.Service
+	// Extended resources can be cluster level or node level.
+	// When set to true, resource fit predicates will ignore absence of extended resources.
+	ignoreMissingExtendedResources bool
```

Comment:
predicateMetadata is not meant to pass configurations to predicates. Our current approach to passing such global configurations it through scheduler arguments. At the very least I would like to see this flag being set from somewhere in scheduler's startup path, so that we can refactor this and put all of such scheduler config in a central location.

Reply:
This flag will be set in the node, which can only use the register function to set the flag.